api/models/test.js

```diff
@@ -7,6 +7,7 @@ let mixin = require('./modelMixin.js');
 let model = Object.assign({
 	title: 'string',
 	questions: [{ref:'Question', type: 'ObjectId'}],
+	hidden: 'string',
```

Comment:
Can you set a default value of `false` in case it is not set by the user in the request.